### PR TITLE
Convert "No parking requirements" checkbox to toggle

### LINKED ISSUE #285: Make no parking requirements a proper toggle
I used a checkmark because it was fast and easy.

Captura de pantalla 2023-10-15 a la(s) 9 38 48 p m

Tony wants a proper toggle though like the original.

Captura de pantalla 2023-10-15 a la(s) 9 38 35 p m

### PR DESCRIPTION
**Pull request by tunglinn**
Resolves #285 

New render:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -214,8 +214,11 @@ <h3>Concept and Initial Report</h3>
     <select multiple class="city-search"></select>
 
     <div class="no-requirements-toggle-container">
-      <label for="no-requirements-toggle">No parking requirements</label>
-      <input id="no-requirements-toggle" type="checkbox" />
+      <span>No Parking Requirements</span>
+      <label class="switch">
```
Comment (Eric-Arellano) [inline index.html:218]: Is it possible for the text to still be in the label? I think that's important for semantic web iirc.

File: src/css/_no-requirements-toggle.scss [hunk 2]
```diff
@@ -8,10 +10,68 @@ $gray: rgb(204, 204, 204);
...
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 34px;
+}
+
+.toggle:before {
```
Comment (Eric-Arellano) [inline src/css/_no-requirements-toggle.scss:52]: Modern best practice is to use two colons for pseudo-elements, whereas `:` is reserved for "states" like `:checked`

```suggestion
.toggle::before {
```

File: src/css/_no-requirements-toggle.scss [hunk 3]
```diff
@@ -8,10 +10,68 @@ $gray: rgb(204, 204, 204);
...
+
+input:checked + .toggle {
+  background-color: #2196f3;
+}
+
+input:focus + .toggle {
+  box-shadow: 0 0 1px #2196f3;
+}
+
+input:checked + .toggle:before {
```
Comment (Eric-Arellano) [inline src/css/_no-requirements-toggle.scss:73]:
```suggestion
input:checked + .toggle::before {
```

**Review by Eric-Arellano — APPROVED** [review]
Yay! Thank you!

**Comment by Eric-Arellano** [comment]
Great, Tony confirmed it's fine for the toggle to be blue rather than the original green. I like your color choice - it looks more modern.

File: index.html (hunk 1 above)
Comment (tunglinn) [reply index.html]: `<label class="switch">` is being used as the oval background of the switch. I switched to a different toggle design that allows the text to be inside the label.

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

**Pull request merged**